modules/peridynamics/include/postprocessors/BondStatusConvergedPostprocessorPD.h

```diff
@@ -0,0 +1,44 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ElementPostprocessor.h"
+#include "PeridynamicsMesh.h"
+
+// Forward Declarations
+
+/**
+ * Postprocessor to check whether the bond status update is converged for a given timestep
```

Comment:
Postprocessor to compute the number of bonds whose status has changed in the most update. Used to check whether the Picard iterations have converged for the current time step.